docs/Using Fleet/Supported-host-operating-systems.md

```diff
@@ -27,6 +25,8 @@ The osquery installer generated for MacOS by `fleetctl package` does not include
 The osquery installer will run on Linux distributions where `glibc` is >= 2.2 (there is ongoing work to make osquery work with `glibc` 2.12+).
 If you aren't sure what version of `glibc` your distribution is using, [DistroWatch](https://distrowatch.com/) is a great resource. 
 
+> On Ubuntu, Fleet Desktop only supports display `0`.
```

Comment:
```suggestion
> On Ubuntu, Fleet Desktop only supports $DISPLAY `:0`.
```

Reply:
this is true for all linux platforms we support

Reply:
Thanks!

Looks like `$DISPLAY` and `:0` are better.

If I Google `$DISPLAY`, this article is the first result: https://askubuntu.com/questions/432255/what-is-the-display-environment-variable